cfy_manager/components/restservice/scripts/create_default_db.sh

```diff
@@ -13,12 +13,15 @@ stage_db_name="stage"
 composer_db_name="composer"
 user=$2
 password=$3
-host=$4
+export PGHOST=$4
```

Comment:
instead of those, we can do `common.sudo(...., env={'PGHOST': '...'})`. That way those exports won't be needed here and also the `if config[POSTGRESQL_CLIENT]['server_username'] == 'postgres':` blocks in the caller are going to be simpler, because they will just need to construct the env dict, instead of having 2 different calls in each if branch

Reply:
Done